Documentation~/modes.md

```diff
@@ -1,42 +1,20 @@
-# Edit modes (Object vs Element)
+# Edit modes and active contexts
 
-ProBuilder uses **Edit modes** to define what you are selecting and editing.
+There are two active contexts when editing in the Unity Editor:
 
-![Editing Modes Example](images/ExampleImage_ObjectAndElementEditingModes.png)
+* The GameObject context, which controls the entire Mesh.
+* The ProBuilder context, which controls the individual elements of the Mesh. The ProBuilder context is further divided into edit modes that define what you are selecting and editing:
```

Comment:
The context doesn't control the mesh. You activate the context to use tools on the mesh.

Reply:
```suggestion
* The ProBuilder context, which allows you to control the individual elements of the Mesh. The ProBuilder context is further divided into edit modes that define what you are selecting and editing:
```

Reply:
Wait, should it be Mesh or mesh?

Reply:
`mesh`. I edited your suggestion.